ios/CozyReactNative/Info.plist

```diff
@@ -144,5 +144,9 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>Pour faire un backup de vos photos</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
```

Comment:
The `Add` version seems to be redundant here, isn't it?

Cf https://developer.apple.com/documentation/bundleresources/information_property_list/nsphotolibraryusagedescription
and https://developer.apple.com/documentation/bundleresources/information_property_list/nsphotolibraryaddusagedescription